manifests/init.pp

```diff
@@ -1,3 +1,4 @@
+# Provides access to Conjur secrets in Puppet manifests
```

Comment:
That's not entirely accurate. It can do that, yes, but there's value in using the class even when you're not using `conjur::secret` (namely, establishing host identity).